packages/types/src/index.d.ts

```diff
@@ -63,7 +63,7 @@ export type ServerConfig = {
   /**
    * Resolves the typescript types for the router and returns an array of types to inject. Uses [`zod-to-ts`](https://github.com/sachinraja/zod-to-ts) by default.
    */
-  resolveTypes?: (router: AnyRouter) => string[] | Promise<string[]>
+  resolveTypes?: (router: AnyRouter) => ResolveTypesReturn | Promise<ResolveTypesReturn>
```

Comment:
Where is the `ResolveTypesReturn` type coming from? I don't see it in this file

Reply:
Should `ResolveTypesReturn` be renamed to something like `ResolvedRouterSchema` since the operation name changed to `getRouterSchema`?

Reply:
Yes but I also don't see it in this file so I don't know where it's being imported from

Reply:
Should be fixed now